control-plane/api-gateway/gatekeeper/gatekeeper_test.go

```diff
@@ -29,6 +29,13 @@ import (
 	"github.com/hashicorp/consul-k8s/control-plane/connect-inject/constants"
 )
 
+const (
+	designatedOpenShiftUIDRange       = "1000700000/100000"
```

Comment:
Is this always the same or does it need to be pulled in from the annotations?

Reply:
In the real world it is pulled from the annotation, for this test we just explicitly set it in the annotation (line 949 for example) since there is no cluster